basic/splunk/src/main/resources/banner.txt

```diff
@@ -0,0 +1,15 @@
+${AnsiColor.GREEN}   _____            _               _____       _                       _   _             
+${AnsiColor.GREEN}  / ____|          (_)             |_   _|     | |                     | | (_)            
+${AnsiColor.GREEN} | (___  _ __  _ __ _ _ __   __ _    | |  _ __ | |_ ___  __ _ _ __ __ _| |_ _  ___  _ __  
+${AnsiColor.GREEN}  \___ \| '_ \| '__| | '_ \ / _` |   | | | '_ \| __/ _ \/ _` | '__/ _` | __| |/ _ \| '_ \ 
+${AnsiColor.GREEN}  ____) | |_) | |  | | | | | (_| |  _| |_| | | | ||  __/ (_| | | | (_| | |_| | (_) | | | |
+${AnsiColor.GREEN} |_____/| .__/|_|  |_|_| |_|\__, | |_____|_| |_|\__\___|\__, |_|  \__,_|\__|_|\___/|_| |_|
+${AnsiColor.GREEN}        | |                  __/ |                       __/ |          ${AnsiColor.BRIGHT_BLUE} ____________          
+${AnsiColor.GREEN}   _____|_|    _            |___/    _____              |___/    _      ${AnsiColor.BRIGHT_BLUE} \ \ \ \ \ \ \                  
+${AnsiColor.GREEN}  / ____|     | |           | |     / ____|                     | |     ${AnsiColor.BRIGHT_BLUE}  \ \ \ \ \ \ \                  
+${AnsiColor.GREEN} | (___  _ __ | |_   _ _ __ | | __ | (___   __ _ _ __ ___  _ __ | | ___ ${AnsiColor.BRIGHT_BLUE}   \ \ \ \ \ \ \                  
+${AnsiColor.GREEN}  \___ \| '_ \| | | | | '_ \| |/ /  \___ \ / _` | '_ ` _ \| '_ \| |/ _ \${AnsiColor.BRIGHT_BLUE}   / / / / / / /                 
+${AnsiColor.GREEN}  ____) | |_) | | |_| | | | |   <   ____) | (_| | | | | | | |_) | |  __/${AnsiColor.BRIGHT_BLUE}  / / / / / / /                 
+${AnsiColor.GREEN} |_____/| .__/|_|\__,_|_| |_|_|\_\ |_____/ \__,_|_| |_| |_| .__/|_|\___|${AnsiColor.BRIGHT_BLUE} / / / / / / /                  
+${AnsiColor.GREEN}  ======| |===============================================| |===========                             
+${AnsiColor.GREEN}        |_|                                               |_|                
```

Comment:
Need to reset color in the end to the `${AnsiColor.DEFAULT}`. Otherwise all my logs are in green. A lot of acid letters :smile: